.vscode/launch.json

```diff
@@ -0,0 +1,84 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch",
+      "type": "coreclr",
+      "request": "launch",
+      "preLaunchTask": "Build",
+      "program": "${workspaceFolder}/DXMainClient/bin/${input:Configuration}/${input:Game}/${input:Engine}/${input:Framework}/win-${input:Arch}/client${input:Suffix}.dll",
+      "args": [],
+      "cwd": "${workspaceFolder}/DXMainClient",
+      "console": "internalConsole",
+      "stopAtEntry": false
+    }
+  ],
+  "inputs": [
+    {
+      "id": "Configuration",
+      "description": "The configuration to build for.",
+      "type": "pickString",
+      "default": "Debug",
+      "options": [
+        "Debug",
+        "Release"
+      ]
+    },
+    {
+      "id": "Game",
+      "description": "The Game to build for.",
+      "type": "pickString",
+      "default": "Ares",
+      "options": [
+        "Ares",
+        "TS",
+        "YR"
+      ]
+    },
+    {
+      "id": "Engine",
+      "description": "The Engine to build for.",
+      "type": "pickString",
+      "default": "WindowsDX",
+      "options": [
+        "UniversalGL",
+        "WindowsDX",
+        "WindowsGL",
+        "WindowsXNA"
+      ]
+    },
+    {
+      "id": "Framework",
+      "description": "The Framework to build for.",
+      "type": "pickString",
+      "default": "net7.0-windows",
+      "options": [
+        "net7.0-windows",
+        "net7.0"
+      ]
+    },
+    {
+      "id": "Arch",
+      "description": "The Arch to build for.",
+      "type": "pickString",
+      "default": "AnyCPU",
+      "options": [
+        "AnyCPU",
+        "x86",
+        "x64",
+        "ARM64"
+      ]
+    },
+    {
+      "id": "Suffix",
+      "description": "The Suffix to run for.",
+      "type": "pickString",
+      "default": "dx",
+      "options": [
+        "dx",
+        "ogl",
+        "xna"
+      ]
+    }
```

Comment:
I am not sure what's the point of those three variables, especially the last one. What's the point of building one thing and running another? For others - they should be tied to selected `Engine` IMO, though those are not as critical as the last one.

Reply:
> I am not sure what's the point of those three variables, especially the last one. What's the point of building one thing and running another? For others - they should be tied to selected `Engine` IMO, though those are not as critical as the last one.

Is there any way to use different values depending on the variable?

Reply:
> Is there any way to use different values depending on the variable?

I am not sure there's a built in way, but you could feed it into an inline console script that matches the correct target, I guess.